server/webapp/WEB-INF/rails.new/app/controllers/api_v1/users_controller.rb

```diff
@@ -43,12 +37,48 @@ def update
       end
     end
 
+    def create
+      result = HttpLocalizedOperationResult.new
+
+      user, created = user_service.withEnableUserMutex do
```

Comment:
Hmm, the `create` method on `UserService` is interesting, I think I should be able to use it.